src/main/java/moe/pgnhd/theshop/Management.java

```diff
@@ -299,4 +299,23 @@ public Seller getSellerFromUser(User user) {
         }
         return null;
     }
+
+    public boolean createSeller(User user){
+        //Will not create a second seller
+
+        if(getSellerFromUser(user) == null){
+            String sql = "INSERT INTO `Seller` \n" +
+                    "(Seller.`id`,Seller.`balance`) \n" +
+                    "VALUES (?, 0);";
+            try(Connection con = ds.getConnection();
+                PreparedStatement stmt = con.prepareStatement(sql)) {
+                stmt.setInt(1, user.getId());
+                stmt.executeUpdate();
+            } catch (SQLException e) {
+                LOG.error(e.getMessage());
+            }
+            return true;
```

Comment:
If execution leaves catch true will be returned without further consequences

Reply:
added return false